recOrder/scripts/cli.py

```diff
@@ -1,40 +1,106 @@
+    )
+
+
+@click.command()
+@click.help_option("-h", "--help")
+@click.argument("filename")
+def info(filename):
+    """View a dataset's basic metadata"""
+    print(f"Reading file:\t {filename}")
+    reader = WaveorderReader(filename)
+    print_reader_info(reader)
+
+
+cli.add_command(info)
+
+
+def print_reader_info(reader):
+    print(f"Positions:\t {reader.get_num_positions()}")
+    print(f"Time points:\t {reader.shape[0]}")
+    print(f"Channels:\t {reader.shape[1]}")
+    print(f"(Z, Y, X):\t {reader.shape[2:]}")
+    print(f"Channel names:\t {reader.channel_names}")
+    print(f"Z step (um):\t {reader.z_step_size}")
+    print("")
 
 
 @click.command()
 @click.help_option("-h", "--help")
-def help():
-    """\033[92mrecOrder: Computational Toolkit for Label-Free Imaging\033[0m
+@click.argument("filename")
+@click.option(
+    "--position",
+    "-p",
+    default=None,
+    multiple=True,
+    help="Integer positions to open. Accepts multiple positions: -p 0 -p 1 -p 10.",
+)
+@click.option(
+    "--layers",
+    "-l",
+    default="position",
+    type=click.Choice(["position", "channel", "p", "c"]),
+    help="Layers as 'position' ('p') or 'channel' ('c')",
+)
+def view(filename, position=None, layers=None):
+    """View a dataset in napari"""
+    print(f"Reading file:\t {filename}")
+    reader = WaveorderReader(filename)
+    print_reader_info(reader)
+
+    if position == ():  # If empty, open all positions
+        position = range(reader.get_num_positions())
+    position = [int(x) for x in position]
+
+    v = napari.Viewer()
+    if layers == "position" or layers == "p":
+        for i in position:
+            try:
+                name = reader.stage_positions[i]["Label"]
+            except:
+                name = "Pos" + str(i)
+            v.add_image(reader.get_zarr(i), name=name)
+            v.layers[-1].reset_contrast_limits()
+        v.dims.axis_labels = ("T", "C", "Z", "Y", "X")
+
+    elif layers == "channel" or layers == "c":
+        print(
+            "WARNING: sending channels to layers is more expensive than sending positions to layers. "
+            "Try loading a small number of positions."
+        )
+
+        ptzyx = (len(position),) + (reader.shape[0],) + reader.shape[2:]
+        for j in range(int(reader.channels)):
+            temp_data = np.zeros(ptzyx)
+            for k, pos in enumerate(position):
+                temp_data[k] = reader.get_array(pos)[:, j, ...]
+            v.add_image(temp_data, name=reader.channel_names[j])
+            v.layers[-1].reset_contrast_limits()
+        v.dims.axis_labels = ("P", "T", "Z", "Y", "X")
 
-    To use recOrder\'s napari plugin, use \033[96mnapari -w recOrder-napari\033[0m
+    napari.run()
 
-    To convert MicroManager .tif files to ome-zarr data format use \033[96mrecOrder.convert\033[0m
 
-    Thank you for using recOrder.
-    """
-    print(help.__doc__)
+cli.add_command(view)
```

Comment:
In the same file, this extra line is equivalent to using `@cli.command()` instead of 
https://github.com/mehta-lab/recOrder/blob/7534ec04ca0b5324abd45e9f734ffd2a212c5730/recOrder/scripts/cli.py#L21

Reply:
Nice! Much improved.